css/style.css

```diff
@@ -121,6 +121,12 @@ body {
     background-color: #FFCB00;
 }
 
+/* BEGIN Table of Contents */
+li.flc-toc-levels-items1, li.flc-toc-levels-items2, li.flc-toc-levels-items3, li.flc-toc-levels-items4 {
+    margin-top: 8px;
+    margin-bottom: 8px;
+}
```

Comment:
"flc-" prefixed classes are only intended to be used for JS selectors. If possible make use of a different CSS selector for styling.

Reply:
The classes for ul and li tags have been pre-defined in this manner. Should I rename the classes for these tags in the HTML code to some other suitable thing to make sure that "flc"- prefix classes are used only for JS selectors?
![Screenshot from 2020-02-28 04-58-43](https://user-images.githubusercontent.com/34926285/75501629-8e277980-59f6-11ea-9498-0113c5927282.png)

Reply:
The ul and li tags have "flc" prefixed class names as shown. Should I rename the classes to something more suitable to make sure the "flc" prefixed classes are only used for JS selectors?
![Screenshot from 2020-02-28 06-53-00](https://user-images.githubusercontent.com/34926285/75501856-2d4c7100-59f7-11ea-8fac-d8da3856dbcb.png)

Reply:
fixed it with f6ec97c53a1f2dec18acb3024bbdb738b4c57030 .